percona/controllers/pgc/users.go

```diff
@@ -36,6 +36,7 @@ func (c *Controller) UpdateUsers(usersSecret *v1.Secret, clusterName, namespace
 }
 
 func updateUserPassword(clientset kubeapi.Interface, username, password string, cluster *crv1.Pgcluster) error {
+	fmt.Println("Update user password for", username, password)
```

Comment:
plain text password in the operator output?

Reply:
removed, thanks https://github.com/percona/percona-postgresql-operator/pull/146/commits/0ad14b7f1c2cc837914914116dc3ec177514d8a8